jenkins-vars.yml

```diff
@@ -16,7 +16,7 @@ repo_vars:
   - PR_DOCKERHUB_IMAGE = 'lspipepr/openssh-server'
   - DIST_IMAGE = 'alpine'
   - DIST_TAG = '3.18'
```

Comment:
Should this be changed as well?

Reply:
It needs removing, we don't actually use it anywhere.